src/julienne/julienne_one_image_prints_s.f90

```diff
@@ -0,0 +1,61 @@
+! Copyright (c) 2024-2025, The Regents of the University of California and Sourcery Institute
+! Terms of use are as specified in LICENSE.txt
+
+submodule(julienne_one_image_prints_m) julienne_one_image_prints_s
+  implicit none
+
+  integer, parameter :: printing_image = 1, me = 1
+
+contains
+
+  module procedure print_string
+#if HAVE_MULTI_IMAGE_SUPPORT
+    my_image: &
+    associate(me => this_image())
```

Comment:
Would really like to see the `this_image` calls use a callback to support Caffeine.

See `ASSERT_PARALLEL_CALLBACKS` code in Assert, [see this code](https://github.com/BerkeleyLab/assert/blob/main/src/assert_m.F90#L40..L60)

Reply:
This has been recorded as issue #57.   Let's defer addressing it until we're closer to merging Caffeine PR [251](https://github.com/BerkeleyLab/caffeine/pull/251).